pkg/assets/mirrors.go

```diff
@@ -40,6 +40,12 @@ var wellKnownMirrors = []mirrorConfig{
 			"https://github.com/kubernetes/kops/releases/download/v{kopsVersion}/",
 		},
 	},
+	{
+		Base: "https://storage.googleapis.com/k8s-artifacts-cni/release/",
```

Comment:
shouldn't we invert this?

long term we'd like to eliminate this host, we have been exiting making kubernetes responsible for hosting third party assets, especially on non-rate-limited hosts that we pay for (And it seems the main use case to not just fetch from github is bypassing their rate limits)

Reply:
ref: https://github.com/kubernetes/k8s.io/issues/7584

Reply:
Agreed, the main use case is bypassing the rate limits.
We started discussing inverting this today, but no conclusion yet. The initial intention was to keep this consistent and decide later which ones to change to the mirror as primary.
@justinsb any preference on how to approach this?

Reply:
> Agreed, the main use case is bypassing the rate limits.

Yeah, the problem is this then makes us responsible for no rate limit, I think if users are pulling so much that github is blocking them, then they probably need to use their own mirror or something?

We have been rate limiting other hosts like registry.k8s.io, but that's not possible here without moving to a different host (at which point ... just use github / official downloads?)